libs/sdk-backend-bear/src/backend/workspace/metadata/index.ts

```diff
@@ -229,13 +229,20 @@ export class BearWorkspaceMetadata implements IWorkspaceMetadata {
         const wrappedAttributeDisplayForms = await this.authCall((sdk) =>
             sdk.md.getObjects<GdcMetadata.IWrappedAttributeDisplayForm>(this.workspace, displayFormUris),
         );
-        return wrappedAttributeDisplayForms.map(
+
+        return wrappedAttributeDisplayForms.reduce(
             (
+                result: IAttributeDisplayFormMetadataObject[],
                 wrappedDisplayForm: GdcMetadata.IWrappedAttributeDisplayForm,
-            ): IAttributeDisplayFormMetadataObject => {
-                const displayFormDetails = wrappedDisplayForm.attributeDisplayForm;
-                return this.buildAttributeDisplayForm(displayFormDetails);
+            ): IAttributeDisplayFormMetadataObject[] => {
+                if (GdcMetadata.isWrappedAttributeDisplayForm(wrappedDisplayForm)) {
```

Comment:
Why reduce/is this check necessary? (Why map is not enough?)

Reply:
wrappedAttributeDisplayForms can include the item isn't `WrappedAttributeDisplayForm`, so I use reduce to return the list include only `WrappedAttributeDisplayForm`